Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
-RUN adduser -S dr4ftuser
```

Comment:
Node images already include a non-root user called `node`, so adding another one is redundant. https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#non-root-user